app/src/lib/stores/updates/changes-state.ts

```diff
@@ -233,12 +233,11 @@ function performEffectsForRebaseStateChange(
   ) {
     const previousTip = prevConflictState.originalBranchTip
 
-    if (
+    const previousTipChanged =
       previousTip !== currentTip &&
       currentBranch === prevConflictState.targetBranch
-    ) {
-      statsStore.recordRebaseSuccessAfterConflicts()
```

Comment:
This check was originally intended to catch rebases where the user completes the rebase outside the app, but I don't think this case is interesting enough to blend it with how the user works within the app flow.

cc @billygriffin for :eyes: and 👍 that we don't care about this case

Reply:
I think that makes sense.

Reply:
yeah i think if we wanted to track this metric, it should be its own measure